src/meta/test/MetaClientTest.cpp

```diff
@@ -503,178 +495,6 @@ TEST(MetaClientTest, HeartbeatTest) {
     ASSERT_EQ(1, ActiveHostsMan::getActiveHosts(sc->kvStore_.get()).size());
 }
 
-class TestMetaService : public cpp2::MetaServiceSvIf {
```

Comment:
Why remove the UTs

Reply:
The UTs are used to test "retry" logic,   maybe we could use another interface instead of "addHosts"

Reply:
Because the `addHosts` interface has been removed, I also deleted the UT here.  And which interface do we want to use instead of it ?

Reply:
For example, you could use "addTag" instead

Reply:
Thx.  After reading the code carefully, I understand the role of those UTs, and have chosen the `heartbeat` instead of `addHosts`.